components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/GraphBasedSequenceHandler.java

```diff
@@ -119,13 +120,16 @@ public void handle(HttpServletRequest request, HttpServletResponse response, Aut
             return;
         }
         if (LoggerUtils.isDiagnosticLogsEnabled()) {
-            Map<String, Object> params = new HashMap<>();
-            params.put(FrameworkConstants.LogConstants.SERVICE_PROVIDER, context.getServiceProviderName());
-            params.put(FrameworkConstants.LogConstants.TENANT_DOMAIN, context.getTenantDomain());
-            LoggerUtils.triggerDiagnosticLogEvent(
-                    FrameworkConstants.LogConstants.AUTHENTICATION_FRAMEWORK, params, LogConstants.SUCCESS,
-                    "Executing script-based authentication",
-                    FrameworkConstants.LogConstants.ActionIDs.HANDLE_AUTH_REQUEST, null);
+            DiagnosticLog.DiagnosticLogBuilder diagnosticLogBuilder = new DiagnosticLog.DiagnosticLogBuilder(
+                    FrameworkConstants.LogConstants.AUTHENTICATION_FRAMEWORK,
+                    FrameworkConstants.LogConstants.ActionIDs.HANDLE_AUTH_REQUEST);
+            diagnosticLogBuilder.inputParam(LogConstants.InputKeys.APPLICATION_NAME, context.getServiceProviderName())
+                    .inputParam(LogConstants.InputKeys.TENANT_DOMAIN, context.getTenantDomain())
+                    .inputParam(LogConstants.InputKeys.CLIENT_ID, context.getSequenceConfig().getApplicationId())
```

Comment:
Shouldn't this `CLIENT_ID` be `APPLICATION ID`?

Reply:
Yes. This should  be appId, clientId should be used in the oauth component

Reply:
This will be fixed with https://github.com/wso2/carbon-identity-framework/pull/4809